core/src/main/java/com/graphhopper/reader/ReaderElement.java

```diff
@@ -145,6 +154,15 @@ public boolean hasTag(List<String> keyList, Set<String> values) {
         return false;
     }
 
+    public boolean hasTagStartsWith(String keyPrefix) {
```

Comment:
According to OSM: "A tag consists of two items, a key and a value", so its a bit unclear what it means when a tag starts with a key prefix. Maybe rename to `hasTagWithKeyPrefix` ?